HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -130,10 +130,25 @@ def customiseFor2018Input(process):
     return process
 
 
+#temporary solution to add GEM geometry for hltGetConfiguration
+def customiseForRun3GEMGeometry34788(process):
+    """Add GEM geometry to output from hltGetConfiguration"""
+    process.GEMGeometryESModule = cms.ESProducer(
+        "GEMGeometryESModule",
+        fromDD4Hep = cms.bool( False ),
+        appendToDataLabel = cms.string( "" ),
+        fromDDD = cms.bool( False )
+    )
+
+    return process
+
 # CMSSW version specific customizations
 def customizeHLTforCMSSW(process, menuType="GRun"):
     
     # add call to action function in proper order: newest last!
     # process = customiseFor12718(process)
 
+    if menuType in ["GRun"]:
```

Comment:
This is problematic - why is this NOT done in general for all production menus (GRun, HIon, PIon, PRef)?

Reply:
As an ES module eventually put into ConfDb, it will neccessarily appear in all of GRun, HIon, PIon, PRef!

Reply:
Do you have evidence that it crashes for HIon, PIon, PRef?

Reply:
I was thinking on that. Since GEM appears in Run-3 only, do we have those menu used for Run-3. 
I mean, I can include it as you suggest. No technical issue on that. Thanks.